.github/workflows/tests.yml

```
@@ -3,22 +3,22 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.7, 3.6]
+    name: Python ${{ matrix.python-version }}
     steps:
-      - uses: actions/checkout@v1
-      - name: Install pyenv
-        uses: gabrielfalcao/pyenv-action@v5
-        with:
-          default: 3.8.2
-          versions: 3.8.2 3.7.7 3.6.10
-      - name: Install dependencies
-        uses: abatilo/actions-poetry@v1.5.0
-        with:
-          python_version: 3.8
-          poetry_version: 1.0.5
-          args: install
-      - name: Run tests
-        uses: abatilo/actions-poetry@v1.5.0
-        with:
-          python_version: 3.8
-          poetry_version: 1.0.5
-          args: run tox
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Poetry
+      run: pip install poetry
+    - name: Install dependencies
+      run: |
+        poetry install
+        poetry add tox-gh-actions
+    - name: Black
+      run: poetry run black .
+    - name: Tox
```

I'd recommend describing what represents this step. Like:

```suggestion
    - name: Tests
```